### PR TITLE
Build script and C code support for the LLVM run-time

### PR DESCRIPTION
**Pull request by housel**

### CODE REVIEW

File: sources/lib/run-time/exceptions.c [hunk 1]
```diff
@@ -53,6 +53,7 @@ MMError dylan_init_thread(void **rReturn, void *(*f)(void *, size_t), void *p, s
 }
 
 
+#ifndef OPEN_DYLAN_BACKEND_LLVM
 void *dylan_callin_handler(void *arg_base, size_t s)
```
Comment (waywardmonkeys) [inline sources/lib/run-time/exceptions.c]: This could be ``#ifdef OPEN_DYLAN_BACKEND_HARP`` ...

File: sources/lib/run-time/llvm-posix-threads.c [hunk 2]
```diff
@@ -0,0 +1,949 @@
...
+  pthread_cond_broadcast(&thread_exit_event);
+  pthread_mutex_unlock(&thread_join_lock);
+
+  return result;
+}
+
+// primitive-make-thread
+dylan_value primitive_make_thread(dylan_value t, dylan_value n, dylan_value p, dylan_value f, DBOOL s)
+{
+  struct KLthreadGYthreadsVdylan *thread
```
Comment (waywardmonkeys) [inline sources/lib/run-time/llvm-posix-threads.c]: This is missing the code that we have in the C run-time for setting the thread name.

It is also missing setting the priority, but we don't do that in the C run-time either. Maybe both could just be marked as TODOs?

Comment (waywardmonkeys) [inline sources/lib/run-time/llvm-posix-threads.c]: Come to think of it, I think we set the thread name elsewhere ... I'd have to look it up.

Comment (housel) [inline sources/lib/run-time/llvm-posix-threads.c]: I'll add TODOs.

**Comment by waywardmonkeys** [comment]
I'll merge this once the *.opendylan.org machines are back online.

**Pull request merged**